packages/config/src/configs/bifrostPolkadot.ts

```diff
@@ -134,6 +134,22 @@ export const bifrostPolkadotConfig = new ChainConfig({
       },
       min: AssetMinBuilder().assetRegistry().currencyMetadatas(),
     }),
+    new AssetConfig({
+      asset: bncs,
+      balance: BalanceBuilder().substrate().tokens().accounts(),
+      destination: moonbeam,
+      destinationFee: {
+        amount: 0.2,
```

Comment:
```suggestion
        amount: 0.05,
```
excessive fee, this value is more correct